src/collections/components/CollectionPartnerAssociationForm/CollectionPartnerAssociationForm.test.tsx

```diff
@@ -290,9 +290,7 @@ describe('The CollectionPartnerAssociationForm component', () => {
     userEvent.clear(blurbField);
     userEvent.type(
       blurbField,
-      'The sky calls to us vastness is bearable ' +
-        'only through love courage of our questions kindling the energy hidden in ' +
-        "matter the only home we've ever known courage of our questions?"
+      'Hmm, this test tends to time out if this line is too long'
     );
```

Comment:
Found the reason why some tests randomly time out - it looks like `userEvent.type()` is rather resource-intensive. Here we're testing that the excerpt has a minimum of 10 characters, so a shorter phrase to speed up this test is more appropriate.

Reply:
weird, but okay.